File: app/src/main/java/com/wwxiaoqi/wechat_backup/bean/CommandResult.java
package com.wwxiaoqi.wechat_backup.bean;

/**
 * 返回的命令结果
 */
public class CommandResult {

  /**
   * 结果码
   **/
  public int result;

  /**
   * 错误信息
   **/
  public String errorMsg;

  /**
   * 成功信息
   **/
  public String successMsg;

  public CommandResult(int result, String successMsg, String errorMsg) {
    this.result = result;
    this.successMsg = successMsg;
    this.errorMsg = errorMsg;
  }

}

File: app/src/main/java/com/wwxiaoqi/wechat_backup/bean/CommandResult.kt
package com.wwxiaoqi.wechat_backup.bean

/**
 * 返回的命令结果
 */
class CommandResult(
  /**
   * 结果码
   */
  var result: Int,
  /**
   * 成功信息
   */
  var successMsg: String?,
  /**
   * 错误信息
   */
  var errorMsg: String?
)